xactserver/src/lib.rs

```diff
@@ -33,3 +34,60 @@ pub enum XsMessage {
     Prepare(proto::PrepareRequest),
     Vote(proto::VoteRequest),
 }
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum XactStatus {
+    Unexecuted,
```

Comment:
Nit: Uncommitted might be a better term? Because execution has already occurred in Postgres. Or maybe "Initialized"

Reply:
These are the states perceived by an `XactState` object. On creation, this object assumes that the transaction is not executed yet, which is true for just-received surrogate transactions. It will get out of this state after the `execute()` method is called. If this is a surrogate transaction, this method executes (validates) it, otherwise, it does nothing.

Reply:
How about "Initialized"?

Reply:
yeah that's good too. I'll change that to `Uninitialized`